spring-integration-core/src/main/java/org/springframework/integration/support/management/graph/LinkNode.java

```diff
@@ -50,7 +50,7 @@ public Type getType() {
 	}
 
 	public enum Type {
-		input, output, error, discard
+		input, output, error, discard, route
```

Comment:
Shouldn't we treat all those route channels as `output` as well?
IMO the visual tool will do the same as for normal `defaultOutputChannel`.
More over it is `default` because we don't match any `mapping`. So, logically all of them are `output`.
I think it won't hurt so much on the client to parse `output` as single element or as an array.

Something similar we have for the `RecipientListRouter`.
Who else ?

CC @aclement